backend/tracim_backend/models/setup_models.py

```diff
@@ -1,26 +1,42 @@
 # import or define all models here to ensure they are attached to the
 # Base.metadata prior to any initialization routines
+import typing
+
+from pyramid.config import Configurator
 from sqlalchemy import engine_from_config
 from sqlalchemy.event import listen
 from sqlalchemy.orm import configure_mappers
 from sqlalchemy.orm import scoped_session
 from sqlalchemy.orm import sessionmaker
 import zope.sqlalchemy
 
+from tracim_backend.lib.utils.utils import sliced_dict
 from tracim_backend.models.auth import Group  # noqa: F401
 from tracim_backend.models.auth import Permission  # noqa: F401
 from tracim_backend.models.auth import User  # noqa: F401
 from tracim_backend.models.data import Content  # noqa: F401
 from tracim_backend.models.data import ContentRevisionRO  # noqa: F401
 from tracim_backend.models.meta import DeclarativeBase  # noqa: F401
 
+if typing.TYPE_CHECKING:
+    from tracim_backend.config import CFG
```

Comment:
why is the import here? if there is a reason, then a comment is required. If no reason, then move it to top of the file

Reply:
explained: 'avoid circular import for type-checking usage of CFG' , done.